packages/client/src/exchanges/postExchangeBookRequests.ts

```diff
@@ -0,0 +1,23 @@
+import { adaptError } from '../helpers/client/formatError.js';
```

Comment:
Same logic as in the `postExchanges` comment. This should be called `postExchangeBookRequest` as it will create a single ExchangeBookRequest. Please, rename this and all other entities related to this to singular (like the type).

Reply:
Same logic as I explained on the `postExchanges` comment.

Reply:
Changed 👍